api/pom.xml

```diff
@@ -243,15 +243,15 @@
 
     <dependencies>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <version>3.0.1</version>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>4.0.2</version>
             <scope>provided</scope>
```

Comment:
This part is not correct. JSP 2.3 depends on Servlet 3.1, not Servlet 4.0 and there is no EE4J Servlet 3.1 release.